internal/pkg/addon/package.go

```diff
@@ -213,6 +215,60 @@ func (t *cfnTemplate) pkg(a *Addons) error {
 	return nil
 }
 
+// packageTransforms searches each node in nodes for the CFN
+// intrinsic function "Fn::Transform" with the "AWS::Include" macro. If it
+// dectects one, and the "Location" parameter is set to a local path, it'll
+// upload those files to S3. If node is a yaml map or sequence, it will
+// recursivly traverse those nodes.
```

Comment:
```suggestion
// recursively traverse those nodes.
```
doing @kohidave proud with the typos

Reply:
😇 i'm just impressed that you catch them